### PR TITLE
Add init command to initialize a `hello world` sample API with GoFr

### PR DESCRIPTION
**Pull request by vipul-rawat**

### CODE REVIEW

File: bootstrap/init.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	Module      string
+	GofrVersion string
+}
+
+func Create(ctx *gofr.Context) (interface{}, error) {
+	name := ctx.Param("name")
+	gofrVersion := ctx.Param("gofr")
+
+	if gofrVersion == "" {
+		gofrVersion = "1.15.0"
```
Comment (Umang01-hash) [inline bootstrap/init.go:47]: Can we use the latest version of GoFr here i.e. `v1.17.0` ?

File: main_test.go [hunk 2]
```diff
@@ -24,3 +19,15 @@ func Test_Main_Version(t *testing.T) {
 
 	assert.Contains(t, out, "dev")
 }
+
+func setArgs(t *testing.T, args ...string) {
+	t.Helper()
+
+	old := os.Args
```
Comment (Umang01-hash) [inline main_test.go:26]: the var name `old` seems to be a bit confusing here, can we rename it to something better?

**Review by Umang01-hash — CHANGES_REQUESTED** [review]

File: bootstrap/init.go (hunk 1 above)
Comment (vipul-rawat) [reply bootstrap/init.go:47]: This defaults to v1.15.0, and we cannot always keep it on par with the latest one, but sure I can change this as long as this PR is in progress.

Comment (Umang01-hash) [reply bootstrap/init.go:47]: Sure! Thankyou

**Review by Umang01-hash — APPROVED** [review]

**Pull request merged**